### DISCUSSION TITLE
XML export overlapping elements on draw.io import

### DISCUSSION THREAD

**Discussion by adaichendt-tv1** [mattieserver/netbox-topology-views#435] [Q&A]
Hey, 

I have a large topology, so the png export produces a rather low quality image. After some research I found about the XML export and subsequent draw.io import. 

Once I import the xml file into draw.io, unfortunately all elements are placed at (0,0). While I can pull them apart manually to get the expected yellow dashed connections, this seems like a tedious task for a network with a couple hundred nodes. 
Is this expected behavior? In the xml file I can see that all nodes are set to (0,0), so I'd expect this is indeed expected. 

Can I tweak this somehow to get an output xml file that includes the coordinates from the live preview? 

Thanks in advance!

**Comment by dreng** [comment]
Hi,

please see #433.
